code/hsec-tools/src/Security/Advisories/Parse.hs

```diff
@@ -82,8 +77,8 @@ data AttributeOverridePolicy
 data ParseAdvisoryError
   = MarkdownError Commonmark.ParseError T.Text
   | MarkdownFormatError T.Text
-  | TomlError TOML.TOMLError T.Text
-  | AdvisoryError TableParseError T.Text
+  | TomlError String T.Text
```

Comment:
It comes with the library change, but personally I see the switch from more specific error constructors to `String` as a regression.  (It's not a show-stopper).

Reply:
I agree that more typed errors are nice in general. My experience with configuration file libraries so far is that no one actually uses them for anything other than rendering them to strings to show the user. Similar to my choice of just using String is that for these simple use cases of getting information out of configuration files, simpler is better.

Of course if my target user base disagrees with my stance I'm open to adapting :)